modules/ixBidAdapter.js

```diff
@@ -311,6 +313,19 @@ function getEidInfo(allEids) {
       }
     }
   }
+
+  const isValidFlocId = utils.deepAccess(flocData, 'id') && utils.deepAccess(flocData, 'version');
+
+  if (isValidFlocId) {
+    var flocEid = {
+      'source': 'chrome.com',
+      'uids': [{'id': flocData.id, 'ext': {'rtiPartner': 'flocId', 'ver': flocData.version}}]
+    };
+
+    toSend.push(flocEid);
+    seenSources['chrome.com'] = 1;
```

Comment:
Any reason for this to be an int instead of bool? Performance difference is probably negligible, but seems like a more appropriate datatype.

Reply:
We are following the existing datatype as [used here](https://github.com/prebid/Prebid.js/blob/d965e83ae64eabfbae7287d4324da3ecda15354c/modules/ixBidAdapter.js#L307) with `seenSources`

Reply:
Ack, I guess that leads to "Why is that object using ints?"
I realize it isn't clearly in the scope of this PR, but it caught my eye and could be a place for improvement if there isn't a good reason for the object to be an int. It looks like this is getting coerced into a boolean for a check[1] anyway.

[1] https://github.com/prebid/Prebid.js/blob/d965e83ae64eabfbae7287d4324da3ecda15354c/modules/ixBidAdapter.js#L359

Reply:
That's a good point and since it's since it's a fairly simple change, I'll make that update